packages/cli/test/dev/utils.js

```diff
@@ -144,7 +144,13 @@ async function exec(directory, args = []) {
 
 async function runNpmInstall(fixturePath) {
   if (await fs.pathExists(join(fixturePath, 'package.json'))) {
-    await execa('yarn', ['install'], {
+    let command;
+    if (await fs.pathExists(join(fixturePath, 'package-lock.json'))) {
+      command = 'npm';
+    } else {
+      command = 'yarn';
```

Comment:
We have some packages that simply _will not_ install and pass tests with `npm`. Letting these be for now as we're updating our test matrix of frameworks currently.

Reply:
Specifically `23-docusaurus` fails to install via npm.

Reply:
Even pining it to 2.0 causes `npm install` timeouts. https://github.com/vercel/vercel/actions/runs/6406652552/job/17391833595?pr=10655